apollo.config.ts

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
use ES6 export
`
export default {content}
`

Reply:
That doesn't work with the apollo command. Changing back to js file